tests/test-spawn.lua

```diff
@@ -10,6 +10,15 @@ local exit_yay, exit_snd = nil, nil
 -- * Using spawn with array is already covered by the test client.
 -- * spawn with startup notification is covered by test-spawn-snid.lua
 
+local tiny_client = function(class)
+    return [[lua -e "Gtk = require('lgi').require('Gtk');
+Gtk.init();window = Gtk.Window { title = 'Client'};
+window:set_wmclass(']]..class.."','".. class..[['); window:show_all();
+Gtk:main{...}"]]
```

Comment:
This program does not ever exit. Especially: It keeps running when the window is closed.